### DISCUSSION TITLE
Retrieving coordinates of peak-to-gene connectivity loops in plotBrowserTrack

### DISCUSSION THREAD

**Discussion by kkobayashikirschvink** [GreenleafLab/ArchR#1061] [Questions / Documentation]
Hi, thank you for developing this wonderful pipeline.

I had a question about the plotBrowserTrack used for plotting Peak2Gene linkages as described in the tutorial here. https://www.archrproject.com/bookdown/peak2genelinkage-with-archr.html

Is there a simple way to retrieve the coordinates of the loops that eventually get plotted in the browser track? Or do I have to retrieve/process it manually from the `getPeak2GeneLinks()` function?

Thank you very much in advance!

**Comment by rcorces** [comment]
Please note that Questions and Feature requests should be posted in the Discussions section. I've migrated your question there.

**Comment by rcorces** [comment]
In theory, `plotBrowserTrack` returns a plot object that could be dissected to get the information you want. But I think using `getPeak2GeneLinks()` using the `returnLoops` parameter is the better solution.
